src/main/java/com/spartronics4915/frc2019/subsystems/MySubsystem.java

```diff
@@ -0,0 +1,28 @@
+public class MySubsystem extends Subsystem
+{
+    private static MySubsystem mInstance = null;
+
+    public static MySubsystem getInstance();
+    {
+        if (mInstance == null)
+        {
+            mInstance = new MySubsystem();
+        }
+        return mInstance;
+    }
+
+    // WantedStates
+
+    // SystemStates
+
+    // Set variables
+
+    // private MySubsystem() - just initiates it
+
+    // private mLoop - put code that does something here
```

Comment:
Good start. Now add a `Loop`, and register that `Loop` in your subsystem's `registerLoops` method.